sdks/python/scripts/generate_pydoc.sh

```diff
@@ -165,7 +165,11 @@ ignore_identifiers = [
   'WindowedTypeConstraint',  # apache_beam.typehints.typehints
 
   # stdlib classes without documentation
-  'unittest.case.TestCase'
+  'unittest.case.TestCase',
+
+  # DoFn param inner classes
+  '_StateDoFnParam',
+  '_TimerDoFnParam'
```

Comment:
Trailing comma for better diffs.

However, I would prefer to see at least some documentation here rather than suppression.

Reply:
Done. As discussed offline, this is due to an issue Sphinx has with extracting docs for these classes.